scripts/linters/other_files_linter.py

```diff
@@ -190,6 +190,15 @@ def check_third_party_libs_type_defs(
 
                 if lib_version[0] == '^':
                     lib_version = lib_version[1:]
+                # In cases where the version is in the form of git commit hashes
+                # such as 'git+https://github.com/username/repo#commit-hash',
+                # we extract the commit hash and use it as the version.
+                elif lib_version[:3] == 'git':
+                    cntr = 0
+                    while cntr < len(lib_version) and lib_version[cntr] != '#':
```

Comment:
I think you can use string functions like .rfind() and .index()

Reply:
@seanlip as per Praneeth's suggestion, should I use regex for checking the string pattern here?

Reply:
Anything that is O(1) and easy for other devs to understand is fine with me.

Reply:
I have pushed the changes with the regex expression. PTAL